apps/web/styles/raqb.css

```diff
@@ -8,18 +8,43 @@
   user-select: none;
 }
 
+.cal-query-builder .group-container {
+  padding: 8px !important;
+  border-radius: 16px;
+  border: 1px solid var(--cal-border-muted);
+  background-color: var(--cal-bg)
+}
+
+.cal-query-builder .group--children:not(:empty) {
+  background-color: var(--cal-bg);
+  border: 1px solid var(--cal-border-muted);
+  border-radius: 10px;
+  padding: 8px;
+}
+
+.cal-query-builder .cal-query-builder-container .group-container {
+  border: none;
+  padding: 0px;
+}
+
+.cal-query-builder .group .group--header {
+  margin-left: 8px;
+}
+
+.group.group-or-rule:has(.group--header, :not(.group--header)):has(.group--children):has(.group--footer) {
```

Comment:
We should add this style as well nested under .cal-query-builder so that we don't accidentally target anything else on the page.